### DISCUSSION TITLE
problem with video adapter cooler

### DISCUSSION THREAD

**Discussion by stasrobeiko** [seerge/g-helper#3451] [General]
I have found one small problem with this wonderful utility. After I enabled Eco mode to disable the discrete graphics card, and then switched back to Standard mode, I noticed that after this procedure, the cooler that cools the discrete graphics somehow stops working, and no matter what the temperature is, even 87 degrees, the cooler does not increase its speed above 30%. To start the cooler, only a system reboot helps. Maybe there is a normal solution to this problem? Or am I doing something wrong?
Zephyrus g14 2021. RTX 3060. Windows 11 24H2

upd:
I read in the discussion that this is a known issue, and it seems to be present on all old G14, G15 or TUF, and there seems to be no solution.
Then a new question: if I just disable the discrete video adapter through the device manager, will this be considered the same as Eco mode? I just have shortcuts on my desktop that let me toggle discrete graphics on and off with a couple of clicks, and that might just be an alternative to Eco mode.

**Comment by Fckgname** [comment]
Today I encountered this problem. After I switched to Eco mode again and returned to Standard mode, the video card cooler started working in standard mode.
I changed modes several times until I realized that the cooler was working again. If you can't do it, you can try to stay in Ultimate mode. The cooler will always work there.

**Reply by stasrobeiko** [reply]
This method doesn't work for me and I don't have Ultimate mode

**Reply by Fckgname** [reply]
Can you give a screenshot of G helper? Ukrain or Ru?

**Reply by stasrobeiko** [reply]
uk

![image](https://github.com/user-attachments/assets/1fa55daa-f210-45b9-b1d2-1a70ea556001)

**Comment by seerge** [comment]
@stasrobeiko hello, 

Fans are always controlled [by firmware / BIOS](https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds) (in case of default fan curves or custom). On some models firmware may have hiccups and misbehave in certain situations.

I see that you use a custom fan curve in the profile, try to use default (click on `Factory Defaults` under `Fans + Power`). This may help.

If it doesn't help - make sure that you have latest BIOS for your model, as Asus often fixes this issues with time.

If you want to override default fan behavior you can try [Experimental build with a manual fan control](https://github.com/seerge/g-helper/discussions/2272)

**Reply by stasrobeiko** [reply]
Using the experimental build with manual fan control really helped get rid of this problem. Thanks for the help
